### PR TITLE
Create /dedicated landing page

### PR DESCRIPTION
**Pull request by Yndira-E**
## Related Issue(s)

https://github.com/FlowFuse/customer/issues/188

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/dedicated.njk [hunk 1]
```diff
@@ -0,0 +1,123 @@
+---
+layout: default
+sitemapPriority: 1.0
+title: FlowFuse Dedicated 
+meta:
+  description: Single-tenant SaaS offering provides operational efficiency and reduced risk for Node-RED applications.
```
Comment (Yndira-E) [inline src/dedicated.njk:6]:
```suggestion
meta:
  title: FlowFuse Dedicated 
  description: Single-tenant SaaS offering provides operational efficiency and reduced risk for Node-RED applications.
```

The title and description must be under `meta` and indented in the same manner

File: src/dedicated.njk [hunk 2]
```diff
@@ -0,0 +1,123 @@
+---
+layout: default
+sitemapPriority: 1.0
```
Comment (Yndira-E) [inline src/dedicated.njk:3]:
```suggestion
```

File: src/dedicated.njk [hunk 3]
```diff
@@ -0,0 +1,122 @@
+---
```
Comment (ZJvandeWeg) [inline src/dedicated.njk:1]: @Yndira-E @Hasmin-AC I thought this was supposed to be under `/product`? So `flowfuse.com/product/dedicated`?

Comment (Yndira-E) [reply src/dedicated.njk:1]: Hi @ZJvandeWeg, according to [the briefing](https://docs.google.com/document/d/1pbNz2JlKwB7GbkUemjvA4gF_qjImV360PXUl6Dj-Tos/edit?usp=sharing) , the specified URL is `flowfuse.com/dedicated`

Comment (Yndira-E) [reply src/dedicated.njk:1]: And I didn’t notice any mention of adding it to the main navigation. Should I proceed to move it and include it under the product menu, as you suggested?

Comment (Hasmin-AC) [reply src/dedicated.njk:1]: In our website discussion we decided that it'll be part of pricing page (similar to gitlab) as it's a type of deployment. 
So not listed under product.

**Pull request merged**